src/pages/user_profile/user_profile.css

```diff
@@ -0,0 +1,26 @@
+.user-stats {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(10rem, 1fr));
+  gap: 1rem;
+}
+.stats-cards {
+  background-color: var(--light-text);
+  border-radius: 4px;
+  padding: 1rem;
+}
+.stats-card-icon {
+  border-radius: 1rem;
+  padding: 0.5rem;
```

Comment:
changes: we can use px for less than 16px

Reply:
Yes will do that, thanks for pointing out